.github/workflows/node.js.yml

```
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x, stable, lts]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
```

```suggestion
      uses: actions/setup-node@v2
```

`@v2` is the latest stable release for `actions/setup-node`. 😉 

See: https://github.com/actions/setup-node/releases

---

Thanks @HonkingGoose,  ignoring my attempt at `stable/lts`, this was the default provided by GitHub, strange they don't use `v2`?